Cards/Animation/CastPosition/CastPos_AllTargets.gd

```diff
@@ -0,0 +1,24 @@
+extends CastPosition
+class_name CastPos_AllTargets
+## CastPos_AllTargets creates an animation for each target
```

Comment:
Yeah CentralPosition chooses a central point between all the targets, so an explosion should work good for this i think.

My intention with the CastPosition is it just creates the animation and sets the initial position. For movement during the animation, we can potentially override the `play_animation` in CastAnimation for a specific animation to give it custom movement. I can add that as a test so we can test that functionality?

Reply:
Yeah if we can test that ! If it doesn't work, it's fine, because we are not even sure we will do that, so let's not try too hard to solve problems we don't have. But if it works out of the box then great